about.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/555b0c0f91.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href=".\styles.css">
+    <!-- Link to bootstrap.min.css-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+    <!-- Link to jquery and popper.min.js -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Link to bootstrap.min.js -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    <title>Interior Design Symposium.</title>
+</head>
+<body>
+  <section class="main-section">
+    <header class="navbar">
+        <nav class="navbar-1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a class="nav-items-11" href="#">English</a>
+            <a class="nav-items-12" href="#">My Page</a>
+            <a class="nav-items-13" href="#">Logout</a>
+        </nav>
```

Comment:
[Optional] For a better user experience on the website, I recommend you modify the page to remove the horizontal scroll.

Your design:
![image](https://user-images.githubusercontent.com/20554583/191714358-2587ffe7-1adc-4832-8bb6-1440337759a4.png)

![image](https://user-images.githubusercontent.com/20554583/191714581-72958c88-0481-4772-ab86-3423ea5a8031.png)